release.json

```diff
@@ -99,4 +99,4 @@
     "dca-1.9.0": {
         "SECURITY_AGENT_POLICIES_VERSION": "v0.3"
     }
-}
```

Comment:
I think we can remove this change

Reply:
Indeed it's fairly useless and probably pulled in more code owners, sorry about that